utils/test_wrapper_thapi_text_pretty.sh.in

```diff
@@ -44,13 +57,26 @@ fi
 
 current_output_bt_text_pretty="${folder_name}/current.bt_text_pretty"
 
-ln -f @top_builddir@/${subdir}/.libs/lib"${be^^}"Interval.so "${folder_name}/"
+test_component=""
+if [ "${subdir}" = "xprof" ]; then
+  ln -f @top_builddir@/${subdir}/.libs/libXAggreg.so "${folder_name}/"
+  ln -f @top_builddir@/${subdir}/.libs/libXTally.so "${folder_name}/"
+  if [ "${be}" = "interval" ]; then
+    test_component=filter.btx_aggreg.aggreg
+  else
+    test_component=sink.btx_tally.tally
+  fi
```

Comment:
```suggestion
  if [ "${be}" = "interval" ]; then
    ln -f @top_builddir@/${subdir}/.libs/libXAggreg.so "${folder_name}/"
    test_component=filter.btx_aggreg.aggreg
  else
    ln -f @top_builddir@/${subdir}/.libs/libXTally.so "${folder_name}/"
    test_component=sink.btx_tally.tally
  fi
```

Reply:
Also maybe just create a new vars for `be` like `tally`?
So we don't need to push nest that in `subdir`